performance-tests/bench_2/worker/SetCftParametersAction.cpp

```diff
@@ -0,0 +1,137 @@
+#include "SetCftParametersAction.h"
+
+#include "MemFunHandler.h"
+
+namespace {
+
+uint32_t one_at_a_time_hash(const uint8_t* key, size_t length) {
```

Comment:
Fine to reuse this, but let's move it to a common header / cpp location (bench2/Common?) so it's not duplicated.